src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -2118,24 +2118,61 @@ groovyscript.wiki.pyrotech.anvil.description=When using hammer or pickaxe it can
+groovyscript.wiki.pyrotech.anvil.inherit.value=Sets if the recipe should be inherited by more advanced machines
+groovyscript.wiki.pyrotech.anvil.add=Adds recipes in the format `name`, `input`, `output`, `hits`, `tier`, `type`, `(optional) inherit`
 
 groovyscript.wiki.pyrotech.barrel.title=Barrel
 groovyscript.wiki.pyrotech.barrel.description=Over time converts a fluid with four items into a new fluid
 groovyscript.wiki.pyrotech.barrel.duration.value=Sets the time required for the recipe to complete
 groovyscript.wiki.pyrotech.barrel.add=Adds recipes in the format `name`, `input1`, `input2`, `input3`, `input4`, `fluidInput`, `fluidOutput`, `duration`
 
+groovyscript.wiki.pyrotech.bloomery.title=Bloomery
+groovyscript.wiki.pyrotech.bloomery.description=Converts item to bloom or a different item by burning it
+groovyscript.wiki.pyrotech.bloomery.add0=Adds recipes in the format `name`, `output`, `input`, `(optional) inherit`
+groovyscript.wiki.pyrotech.bloomery.add1=Adds recipes in the format `name`, `output`, `input`, `burnTime`, `(optional) inherit`
+groovyscript.wiki.pyrotech.bloomery.addBloom0=Adds recipes in the format `name`, `bloomOutput`, `input`, `(optional) inherit`
+groovyscript.wiki.pyrotech.bloomery.addBloom1=Adds recipes in the format `name`, `bloomOutput`, `minYield`, `maxYield`, `input`, `burnTime`, `experience`, `(optional) inherit`, `failureChance`, `failureItems`
+groovyscript.wiki.pyrotech.bloomery.slag.value=Sets the slag item
+groovyscript.wiki.pyrotech.bloomery.bloom.value=Sets the actual output (bloom) of the Bloomery. Setting langKey, anvil values, experience, bloom yields and failure values are unnecessary if this is set
+groovyscript.wiki.pyrotech.bloomery.inherit.value=Sets if the recipe should be inherited by more advanced machines
+groovyscript.wiki.pyrotech.bloomery.langKey.value=Sets the localization key to use for blooms name
+groovyscript.wiki.pyrotech.bloomery.anvilHit.value=Sets how often the bloom needs to be hit on the anvil
+groovyscript.wiki.pyrotech.bloomery.burnTime.value=Sets the time required for the recipe to complete
+groovyscript.wiki.pyrotech.bloomery.anvilType.value=Sets the tool that needs to be used in anvil
+groovyscript.wiki.pyrotech.bloomery.anvilTiers.value=Sets the allowed anvil tiers bloom allows
+groovyscript.wiki.pyrotech.bloomery.experience.value=Sets the experience given from bloom
+groovyscript.wiki.pyrotech.bloomery.bloomYieldMin.value=Sets minimum amount of drops bloom can give
+groovyscript.wiki.pyrotech.bloomery.bloomYieldMax.value=Sets maximum amount of drops bloom can give
+groovyscript.wiki.pyrotech.bloomery.failureChance.value=Sets the chance for bloom to give failure item instead of the actual output
+groovyscript.wiki.pyrotech.bloomery.failureOutput.value=Sets the output to drop on failure
+
+groovyscript.wiki.pyrotech.brick_crucible.title=Refractory Crucible
+groovyscript.wiki.pyrotech.brick_crucible.description=Melts an item into liquid.
```

Comment:
this could imply that there is a map of `<Item, Fluid>`, and this merely converts from the former to the latter instead of being an actual recipe - due to that, i think `Converts an input itemstack into an output fluidstack after a duration` would be preferable.